lib/std/os/linux.zig

```diff
@@ -3593,11 +3593,12 @@ pub const dl_phdr_info = extern struct {
 };
 
 pub const CPU_SETSIZE = 128;
```

Comment:
`man 3 CPU_SET` shows me:
```
The first CPU on the system corresponds to a cpu value of 0, the next CPU corresponds to a cpu value of 1, and       so on.  No assumptions should be made about particular CPUs being available, or the set of CPUs being contigu‐
       ous, since CPUs can be taken offline dynamically or be otherwise absent.  The constant CPU_SETSIZE  (currently
       1024) specifies a value one greater than the maximum CPU number that can be stored in cpu_set_t.
```

- 1. This should have a comment on what it does
- 2. This should be corrected, because Linux apparently supports at maximum 1024 CPUs.